recipes/limits/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "limits" %}
+{% set version = "2.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: da6346f0dcf85f17f0f1cc709c3408a3058cf6fee68313c288127c287237b411
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python >=3.7
+  run:
+    - python >=3.7
+    - setuptools
```

It looks like setuptools is only needed at package build time (running `setup.py`), not at run time:
```suggestion
  host:
    - pip
    - setuptools
    - python >=3.7
  run:
    - python >=3.7
```